packages/edit-post/src/components/editor-regions/index.js

```diff
@@ -33,25 +33,25 @@ function EditorRegions( { footer, header, sidebar, content, publish, className }
 				>
 					{ content }
 				</div>
-				{ !! publish && (
+				{ !! sidebar && (
 					<div
-						className="edit-post-editor-regions__publish"
+						className="edit-post-editor-regions__sidebar"
 						role="region"
-						/* translators: accessibility text for the publish landmark region. */
-						aria-label={ __( 'Editor publish' ) }
+						aria-label={ 'Editor settings' }
```

Comment:
For consistency, you might want to add a comment for translators here as well.

Reply:
Good catch, it should be wrapped with the translate helper function as well 👍

Reply:
Ah, looks like it wasn't there before.